lua/arrow/buffer_ui.lua

```diff
@@ -219,6 +230,8 @@ function M.spawn_action_windows(call_buffer, bookmarks, line_nr, col_nr, call_wi
 end
 
 function M.openMenu()
+	persist.update()
+	persist.sync_buffer_bookmarks()
```

Comment:
We only update line position when 1. bufleave 2. opemenu 3. vimleave, which is really clean thanks to arrow's single ui.